src/api/middlewares/isAuth.ts

```
@@ -7,21 +7,21 @@ import config from '../../config';
  */
-const getTokenFromHeader = (req) => {
+const getTokenFromHeader = (req: any) => {
   const { authorization } = req.headers;
   if (
     (authorization && authorization.split(' ')[0] === 'Token') ||
     (authorization && authorization.split(' ')[0] === 'Bearer')
   ) {
     return authorization.split(' ')[1];
   }
-  // tslint:disable-next-line: no-null-keyword
+  // eslint-disable-next-line @typescript-eslint/no-null-keyword
```

ESLint

[error] Definition for rule '@typescript-eslint/no-null-keyword' was not found.

(@typescript-eslint/no-null-keyword)